pkg/oidc/response_template.go

```diff
@@ -0,0 +1,35 @@
+package oidc
+
+const responseTpl = `
+{{ $title := "Authentication Succeeded" }}
+{{ if .Error }}
+	{{ $title = "Authentication Failed" }}
+{{ end }}
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>{{ $title }}</title>
+	</head>
+	<body>
+		<h1>{{ $title }}</h1>
+	{{ if .Error }}
+		<h3>Try again after correcting the error below</h3>
+		<hr>
+		<dl>
+			<dt>Error</dt>
+			<dd>{{.Error}}</dd>
```

Comment:
Are we sure we want to leak info about the failure?

Reply:
I don't wanna call it leak, since those are all client side messages. We probably wanna log errors somewhere in the local machine, for debug/tracing purpose.